static/src/styles/core/text.styl

```diff
@@ -51,3 +51,12 @@ p, ol, ul {
   margin-top: 0;
   margin-bottom: $Text__baseSize;
 }
+
+.Text--sectionHeading {
+  font-family: $Text__primaryFont;
+  text-transform: uppercase;
+  letter-spacing: 0.15em;
+  font-weight: 500;
+  Text__headerScale(0);
+  color: $Color__mediumGrayText;
```

Comment:
This colour doesn't exist here but does in another PR so this is in prep for that going in